### PR TITLE
Bug fix - modify button alignment for my project DP-600

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class ProgramListComponent implements OnInit {
 
   ngOnInit() {
     this.checkIfUserIsContributor();
-    this.issourcingOrgAdmin = this.userService.isSourcingOrgAdmin();
+    this.issourcingOrgAdmin = this.userService.isSourcingOrgAdmin() && this.router.url.includes('/sourcing');
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.ts:54]: @Ekta29yadav if you want to apply this.router.url.includes('/sourcing'); this check, apply on the specific code, do not modify this.

Comment (Ekta29yadav) [reply src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.ts]: Updated the code

File: src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html [hunk 2]
```diff
@@ -167,11 +167,11 @@ <h4 class="sb-pageSection-title">
                 </td>
                 <td class="text-center pr-10">
 
-                  <div *ngIf="activeMyProgramsMenu && program.nomination_status  == 'Pending' && canNominate(program)" (click)="getProjectToModify(program)" class="d-inline-block cursor-pointer pr-10 va-text-sub">
+                  <div *ngIf="activeMyProgramsMenu && program.nomination_status  == 'Pending' && canNominate(program)" (click)="getProjectToModify(program)" class="d-inline-block cursor-pointer pr-10 pt-5">
                     <i class="icon edit sb-color-primary ui fs-0-92"></i>
                     <span class="sb-color-primary">{{ resourceService?.frmelmnts?.lbl?.modifyNomination.button }}</span>
                   </div>
-                  <span [ngClass]="{'pull-right': !issourcingOrgAdmin}">
+                  <span [ngClass]="{'pull-right': !issourcingOrg}">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html:174]: why added extra variable? here you could have added !(issourcingOrgAdmin && router.url.includes('/sourcing'))

Comment (twsvaishali) [reply src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html:174]: @Ekta29yadav

Comment (Ekta29yadav) [reply src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html]: Updated the PR

**Pull request merged**